opea-comps/mega-service/app.py

```
@@ -9,6 +10,12 @@
 from comps.cores.mega.constants import ServiceType, ServiceRoleType
 from comps import MicroService, ServiceOrchestrator
 import os
+import aiohttp
+from comps.cores.mega.utils import handle_message
```

Ruff (0.8.2)

`comps.cores.mega.utils.handle_message` imported but unused

Remove unused import: `comps.cores.mega.utils.handle_message`

(F401)